sdk/program/src/keccak.rs

```diff
@@ -2,127 +2,12 @@
 //!
 //! [keccak]: https://keccak.team/keccak.html
 
-use {
-    crate::sanitize::Sanitize,
-    borsh::{BorshDeserialize, BorshSchema, BorshSerialize},
-    sha3::{Digest, Keccak256},
-    std::{convert::TryFrom, fmt, mem, str::FromStr},
-    thiserror::Error,
-};
+use sha3::Keccak256;
```

Comment:
nit: can you fold all the use statements together? If you run `./cargo nightly fmt --all` from the root of the repo it should take care of it automatically